dags/datasets/datastore-resource-description.py

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
I should have mentioned this before - please move this file to `dags/sustainment` since its not an ETL job

Reply:
ok